decentdao/fractal-interface
# Replacing Rainbowkit with Web3Modal

This PR contains following changes:
- Replace `RainbowKit` with `Web3Modal`
- Add HTTPS support for local dev server in order to overcome possible issues for local development
- Implement trick to open web3 modal from 2nd attempt if 1st attempt fails (happens frequently both on local and deployed site)

Closes #1450 

Moving further, we'll have to update `web3Modal` version to `v4.x+` together with other web3 libraries (viem, wagmi, etc)

## Testing

1. Connect your wallet through different wallet providers (MetaMask, CoinBase, Injected Wallet, WalletConnect)
2. Try to create any type of Safe (better to test with Multisig and Azorius ERC-20/ERC-721). Make sure safe is properly created
3. Try to create proposal for that Safe(s). Make sure proposal properly created
4. Try to vote on that proposal. Make sure vote appears right after you voted, breakdown updated properly, etc.
5. Try to execute that proposal
